src/react/useQueries.ts

```diff
@@ -68,7 +68,7 @@ type GetResults<T> =
 /**
  * QueriesOptions reducer recursively unwraps function arguments to infer/enforce type param
  */
-type QueriesOptions<
+export type QueriesOptions<
```

Comment:
can we export the `QueriesResults` too please?

Reply:
@TkDodo All set!